scripts/systemd/check_and_upgrade_serial_keel.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+# This script will either setup a fresh serial-keel service instance, or upgrade your existing one.
+# This will also be executed during every bootup, so a check for an upgrade will be performed.
+set -e
+
+CURRENT_USER=$(whoami)
+USER_HOME="/home/$CURRENT_USER/"
+
+mkdir -p $USER_HOME/.config/systemd/user/
+
+# The name of the branch can be specified as a argument, or it defaults to 'main'
+BRANCH=${1-main}
+CONFIG_PATH=${2-/home/$CURRENT_USER/config.ron}
```

Comment:
You can reuse the home var here.

Reply:
Added a commit that resolves the remaining comments.